probe/awsecs/client.go

```diff
@@ -70,16 +70,19 @@ type EcsInfo struct {
 	TaskServiceMap map[string]string
 }
 
-func newClient(cluster string, cacheSize int, cacheExpiry time.Duration) (EcsClient, error) {
+func newClient(cluster string, cacheSize int, cacheExpiry time.Duration, clusterRegion string) (EcsClient, error) {
 	sess := session.New()
-
-	region, err := ec2metadata.New(sess).Region()
-	if err != nil {
-		return nil, err
+  var err error
```

Comment:
style: this line isn't correctly indented. Make sure all code has passed through `gofmt -s`

Reply:
Done for all files